src/components/DetailsDrillDownAlert/DetailsAlertsMetrics.js

```diff
@@ -0,0 +1,188 @@
+/*
+Copyright 2019 Iguazio Systems Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License") with
+an addition restriction as set forth herein. You may not use this
+file except in compliance with the License. You may obtain a copy of
+the License at http://www.apache.org/licenses/LICENSE-2.0.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License.
+
+In addition, you may not use the software for any purposes that are
+illegal under applicable law, and the grant of the foregoing license
+under the Apache 2.0 license is conditioned upon your compliance with
+such restriction.
+*/
+import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react'
+import PropTypes from 'prop-types'
+import { useDispatch, useSelector } from 'react-redux'
+import { isEmpty } from 'lodash'
+
+import ApplicationMetricCard from '../DetailsMetrics/ApplicationMetricCard'
+import DatePicker from '../../common/DatePicker/DatePicker'
+import NoData from '../../common/NoData/NoData'
+import NoMetricData from '../DetailsMetrics/NoMetricData'
+
+import { REQUEST_CANCELED } from '../../constants'
+import detailsActions from '../../actions/details'
+import modelEndpointsActions from '../../actions/modelEndpoints'
+import { groupMetricByApplication } from '../../elements/MetricsSelector/metricsSelector.util'
+
+import {
+  datePickerPastOptions,
+  PAST_24_HOUR_DATE_OPTION,
+  TIME_FRAME_LIMITS
+} from '../../utils/datePicker.util'
+
+const DetailsAlertsMetrics = ({ selectedItem }) => {
+  const [metrics, setMetrics] = useState([])
+  const [requestErrorMessage, setRequestErrorMessage] = useState('')
+  const metricsContainerRef = useRef(null)
+  const metricsValuesAbortController = useRef(new AbortController())
+  const prevSelectedEndPointNameRef = useRef('')
+  const [metricOptionsAreLoaded, setMetricOptionsAreLoaded] = useState(false)
```

Comment:
can we check from the store if we  have metrics?

Reply:
we can check with: "detailsStore.metricsOptions.all" or with generatedMetrics.
i added validation with generatedMetrics is empty, in this case will be message "Metrics data not found".

Reply:
so we can remove this state?

Reply:
will be removed in ML-8106